src/components/notification-panel/notification-item.js

```diff
@@ -54,6 +54,11 @@ const NotificationMessage = styled.div`
   p {
     margin-top: 0;
   }
+  a {
```

Comment:
prefer not to use `!Important` it will be hard to override. you can add a class before a
```
styled.div`
  .notification-link {
      a {
          color: white;
      }
  }
`
```